src/chats/components/secondaryBar/ExpandedSidebarListItem.tsx

```diff
@@ -168,8 +168,7 @@ const ExpandedSidebarListItem: React.FC<ExpandedSidebarListItemProps> = ({ roomI
 				case MessageType.CONFIGURATION_MSG:
 					return <ConfigurationMessageLabel message={lastMessageOfRoom} />;
 				default:
-					console.error('Message to replace: ', lastMessageOfRoom.type);
-					return 'Message to replace';
+					return '';
```

Comment:
Here keep the error log otherwise we don't now if there is a case not handled or an error happens

Reply:
The default branch is reached when the last message type is DATE so the error case was already been fixed by https://github.com/zextras/carbonio-ws-collaboration-ui/pull/251 .